pypeit/tests/test_biasframe.py

```diff
@@ -93,14 +93,13 @@ def test_run_and_master(kast_blue_bias_files):
     bias_frame2 = biasframe.BiasFrame('shane_kast_blue', setup=setup, master_dir=master_dir,
                                       mode='reuse')
     bias2 = bias_frame2.master()
-    assert isinstance(bias_frame2.stack, np.ndarray)
+    assert isinstance(bias_frame2.msframe, np.ndarray)
```

Comment:
Does this mean `stack` is empty in this case?
If so, I'd modify the code to point `stack` at `msframe`
otherwise other things may break down the line.

Reply:
Yes, stack is empty.